imports/plugins/included/taxes-avalara/client/accounts/exemption.js

```diff
@@ -6,11 +6,16 @@ import { Packages, Accounts } from "/lib/collections";
 import { Accounts as AccountsSchema } from "/lib/collections/schemas/accounts";
 import { TaxEntityCodes } from "/client/collections";
 
+let entityCodeList = [];
+let currentAccount;
+
 Template.taxSettingsPanel.helpers({
   account() {
     const sub = Meteor.subscribe("UserAccount", this.member.userId);
```

Comment:
Just noticing this now, but should thus subscription be in the helper? Seems like it should be in an `onCreated` event?